src/dialects/oracle/data-types.js

```diff
@@ -0,0 +1,470 @@
+// Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved
+
+'use strict';
+
+const moment = require('moment');
+const momentTz = require('moment-timezone');
+
+module.exports = BaseTypes => {
+  const warn = BaseTypes.ABSTRACT.warn.bind(
+    undefined,
+    'https://docs.oracle.com/database/122/SQLRF/Data-Types.htm#SQLRF30020'
```

Comment:
Always use the redirecting URL to get the latest doc: https://www.oracle.com/pls/topic/lookup?ctx=dblatest&id=GUID-7B72E154-677A-4342-A1EA-C74C1EA928E6

Reply:
Done